forms-flow-web/src/containers/MessageModal.jsx

```diff
@@ -9,10 +9,12 @@ const MessageModal = React.memo((props) => {
   return (
     <>
       <Modal show={modalOpen}>
```

Comment:
do you think we should add `onHide={onConfirm}` here? It will allow user to close popup by click anywhere on backdrop and do not need to reach to "ok" button.

Reply:
That's a good one. I'll add it.